tests/integration/test_msgraph/test_API/test_API.py

```diff
@@ -0,0 +1,530 @@
+t1_configurations = load_configuration_template(configs_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
+
+# Test configurations
+t2_cases_path = Path(TEST_CASES_PATH, 'cases_future_no.yaml')
+t2_configuration_parameters, t2_configuration_metadata, t2_case_ids = get_test_cases_data(t2_cases_path)
+t2_configurations = load_configuration_template(configs_path, t2_configuration_parameters,
+                                                t2_configuration_metadata)
+
+# ---------------------------------------------------- TEST_CURL_SIZE ---------------------------------------------------
+# Test configurations
+t3_cases_path = Path(TEST_CASES_PATH, 'cases_curl_size.yaml')
+t3_configuration_parameters, t3_configuration_metadata, t3_case_ids = get_test_cases_data(t3_cases_path)
+t3_configurations = load_configuration_template(configs_path, t3_configuration_parameters,
+                                                t3_configuration_metadata)
+
+# ---------------------------------------------------- TEST_RESOURCES ---------------------------------------------------
+# Test configurations
+t4_cases_path = Path(TEST_CASES_PATH, 'cases_valid_resource.yaml')
+t4_configuration_parameters, t4_configuration_metadata, t4_case_ids = get_test_cases_data(t4_cases_path)
+t4_configurations = load_configuration_template(configs_path, t4_configuration_parameters,
+                                                t4_configuration_metadata)
+
+# Test configurations
+t5_cases_path = Path(TEST_CASES_PATH, 'cases_invalid_resource.yaml')
+t5_configuration_parameters, t5_configuration_metadata, t5_case_ids = get_test_cases_data(t5_cases_path)
+t5_configurations = load_configuration_template(configs_path, t5_configuration_parameters,
+                                                t5_configuration_metadata)
+
+# ---------------------------------------------------- TEST_AUTH_TOKEN ---------------------------------------------------
+# Test configurations
+t6_cases_path = Path(TEST_CASES_PATH, 'cases_valid_auth.yaml')
+t6_configuration_parameters, t6_configuration_metadata, t6_case_ids = get_test_cases_data(t6_cases_path)
+t6_configurations = load_configuration_template(configs_path, t6_configuration_parameters,
+                                                t6_configuration_metadata)
+
+# Test configurations
+t7_cases_path = Path(TEST_CASES_PATH, 'cases_invalid_auth.yaml')
+t7_configuration_parameters, t7_configuration_metadata, t7_case_ids = get_test_cases_data(t7_cases_path)
+t7_configurations = load_configuration_template(configs_path, t7_configuration_parameters,
+                                                t7_configuration_metadata)
+
+# Test configurations.
+daemons_handler_configuration = {'all_daemons': True, 'ignore_errors': True}
+local_internal_options = {MODULESD_DEBUG: '2'}
+
+
+@pytest.fixture(scope="session")
```

Comment:
All fixtures should be inside a `conftest.py` file. Move them to 
`tests/integration/test_msgraph/conftest.py`